src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/naxxramas/instance_naxxramas.cpp

```diff
@@ -22,6 +22,7 @@ SDCategory: Naxxramas
 EndScriptData */
 
 #include "AI/ScriptDevAI/include/sc_common.h"
+#include "AI/ScriptDevAI/include/sc_instance.h"
```

Comment:
sorry about this bit... my dang IDE automatically adds includes it thinks it needs... this part can be omitted